app/src/main/java/de/westnordost/streetcomplete/about/CreditsFragment.kt

```diff
@@ -71,8 +72,12 @@ class CreditsFragment : Fragment(R.layout.fragment_credits) {
     }
 
     private suspend fun readCodeContributors() = withContext(Dispatchers.IO) {
-        resources.getYamlObject<List<String>>(R.raw.credits_code).map(::withLinkToGithubAccount) +
-            getString(R.string.credits_and_more)
+        // because they are mentioned as "main contributors" already
+        val skipUsers = setOf("westnordost", "FloEdelmann", "matkoniecz")
+        resources.getYamlObject<List<Contributor>>(R.raw.credits_contributors)
```

Comment:
But it is a JSON file?

Reply:
yes

Reply:
Interesting that parsing JSON files as YAML seems to work fine then...

But why not write a YAML file directly?

Reply:
No real reason

Reply:
ok